### PR TITLE
Adds and updates validations

### PR DESCRIPTION
**Pull request by hoenth**
Adds additional validations and updates some of the existing validations to conform more with rails standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -728,10 +728,13 @@ You can create validations on your objects just like Rails' built in ActiveModel
 
 ```ruby
 class Person < Flexirest::Base
-  validates :first_name, presence:true
-  validates :password, length:{within:6..12}
-  validates :post_code, length:{minimum:6, maximum:8}
-  validates :salary, numericality:true, minimum:20_000, maximum:50_000
+  validates :first_name, presence: true #ensures that the value is present and not blank
+  validates :last_name, presence: true #ensures that the value is non-nil only
```
Comment (andyjeffries) [inline README.md]: The comment seems to suggest they have different behaviours but the same key?

**Comment by andyjeffries** [comment]
Aside from that one comment, I'm happy to merge it.

**Comment by hoenth** [comment]
Just pushed a fix for the README typo

**Comment by andyjeffries** [comment]
Merged, many thanks.  Do you need a new version release for your project or are you happy using the upstream github URL?

**Comment by hoenth** [comment]
I can use the the github url. But if you want to push a new version, that is ok too. 

Best, 
Tom

> On Nov 20, 2015, at 8:27 AM, Andy Jeffries <notifications@github.com> wrote:
> 
> Merged, many thanks. Do you need a new version release for your project or are you happy using the upstream github URL?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/andyjeffries/flexirest/pull/2#issuecomment-158403101>.
> 

Tom Hoen
VP Customer Facing Technology
Network for Good
410-215-0905
tom.hoen@networkforgood.com
Unleashing Generosity
Creating fundraising software that delights

**Pull request merged**